contracts/xchainGauges/AnyCallTranslator.sol

```diff
@@ -88,17 +103,28 @@ contract AnyCallTranslator is OwnableUpgradeable {
         }
     }
 
+    /// @notice Set the AnyCall contract address
+    /// @dev Only callable by owner
+    /// @param _anyCallContract address of the AnyCallV6 contract
     function setAnyCall(address _anyCallContract) external onlyOwner {
         anyCallContract = _anyCallContract;
         anyCallExecutor = ICallProxy(_anyCallContract).executor();
     }
 
+    /// @notice withdraw any ETH that was sent to AnyCall contract
+    /// @dev Only callable by owner
+    /// @param _amount amount of ETH to withdraw
     function withdraw(uint256 _amount) external onlyOwner {
```

Comment:
How would we get the ETH out of this contract, once we've gotten it out of the AnyCall contract?

Reply:
`rescue` function can be used for ETH withdraw from this contract.